lib/api.js

```diff
@@ -28,8 +28,23 @@ class API {
     }
 
     if (this.type === 'Module') {
+      const subObjects = {}
       this.events = new Collection('Event', this, '#events', 2)
-      this.methods = new Collection('Method', this, '#methods', 2)
+      this.properties = new Collection('Property', this, `#properties`, 2)
+      this.methods = new Collection('Method', this, '#methods', 2).filter(method => {
```

Comment:
Here we scan through all methods and if any of them have a "superObject" we remove them from the methods list and put them in the properties list under a new "Object" property with the same name as the given "superObject"